examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -18,60 +18,13 @@
  */
 
 const app = {
-  isInitialized: false,
```

Comment:
this is from #183, but it's no longer needed now that we actually have a callback for this